etc/alignak.cfg

```diff
@@ -87,28 +164,30 @@ no_event_handlers_during_downtimes=1
 # [Optionnal], a pack distribution file is a local file near the arbiter
 # that will keep host pack id association, and so push same host on the same
 # scheduler if possible between restarts.
-pack_distribution_file=/var/lib/alignak/pack_distribution.dat
+pack_distribution_file=/usr/local/var/lib/alignak/pack_distribution.dat
 
 
 
-## Arbiter daemon part, similar to ini
+# --------------------------------------------------------------------
+## Arbiter daemon part, similar to daemon ini file
+# --------------------------------------------------------------------
 
 #If not specified will use lockfile direname
-workdir=/var/run/alignak
+workdir=/usr/local/var/run/alignak
 
 # Lock file (with pid) for Arbiterd
-lock_file=/var/run/alignak/arbiterd.pid
+lock_file=/usr/local/var/run/alignak/arbiterd.pid
 
 # The arbiter can have it's own local log
-local_log=/var/log/alignak/arbiterd.log
+local_log=/usr/local/var/log/alignak/arbiterd.log
 
 # Accepted log level values: DEBUG,INFO,WARNING,ERROR,CRITICAL
 #log_level=WARNING
 
 # User that will be used by the arbiter.
 # If commented, run as current user (root?)
-#alignak_user=alignak
-#alignak_group=alignak
+alignak_user=alignak
```

Comment:
You are right ... but I did not changed anything there ... the arbiter loads this :) We should use `user` and `group` like the other daemons ... I log an issue for this